### ISSUE TITLE
`Unresolved type undefined` instead of unresolved type name

### ISSUE THREAD

**Issue by ojab** [castwide/solargraph#398]
```ruby
# @param tz [TimeZone]
def xxx(tz:)
end
```

```
/tmp/sg/as.rb:2 - Unresolved type undefined for tz param on #xxx
1 problem found.
```

It would be more useful to have `Unresolved type TimeZone for tz param on #xxx` message to know which type is not resolved without reading the code.

**Comment by castwide** [comment]
Confirmed. I'll fix it in an upcoming release.

**Comment by castwide** [comment]
Fixed in v0.40.3.
